### DISCUSSION TITLE
Payload logging

### DISCUSSION THREAD

**Discussion by seunghyunnn** [bitcraze/discussions#200] [Q&A]
Hi, 

I am currently trying to collect the data with the crazyflie. 
While I am using the attributes for logging such as stateEstimate.x, stateEstimate.y, pm.vbat, etc. for every 0.1 seconds, I also want to log the payload - the total weight of a crazyflie is carrying. 
(Currently, the crazyflie first hovers carrying the paper clips and drops the paper clips while it is flying. I want to see the change in battery consumption of crazyflie associated with its change in the payload.) 

Does the crazyflie have a mechanism to measure the real-time payload it is carrying? If it has, is there any logging attributes/API given to measure and record it? 

Thank you so much!

**Comment by krichardsson** [comment]
Sorry, there is no such functionality. In theory you could possibly derive it from the thrust that is used when hovering, but my guess is that it will be tricky (it would be interesting to try though).

**Comment by seunghyunnn** [comment]
Oh I see. Thank you so much!
